.github/workflows/test.yml

```diff
@@ -37,7 +37,7 @@ jobs:
             ${{ runner.os }}-
 
       - name: Install dependencies
-        run: yarn install --network-timeout 1000000
+        run: npm install
```

Comment:
just wondering, why did u decide to use npm instead of yarn? IIRC, yarn has a better performance in downloading due to caching

Reply:
hmm i didn't consider that. I wanted to use `npm` instead of yarn initially because I wanted one less barrier of entry to our codebase (ie not having people to install yarn before developing for our app). And for that reason, I wanted our github workflows to be consistent about that as well.

Reply:
I see, noted on this